utils/keys.go

```diff
@@ -103,11 +104,52 @@ func ImportKeys(from io.Reader, to []Importer) error {
 		toWrite []byte
 	)
 	for block, rest := pem.Decode(data); block != nil; block, rest = pem.Decode(rest) {
+		if block.Headers["role"] == "" {
+			// no worries about if check as for GUN here because empty roles will get a role:notary.DefaultImportRole
+			block.Headers["role"] = fallbackRole
+		}
+		if rawPath := block.Headers["path"]; rawPath != "" {
+			pathWOFileName := strings.TrimSuffix(rawPath, filepath.Base(rawPath))
+			if strings.Contains(pathWOFileName, notary.NonRootKeysSubdir) {
```

Comment:
can we use `strings.HasPrefix` here instead of `strings.Contains`?

Also, we can include slashes in the `TrimSuffix` and `TrimPrefix` instead of having to slice when assigning to the PEM header